docusaurus/docs-internals/editor-controls/ui-controls/button.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 1
+---
+
+# Button
+
+Brizy's control button features a panel button that can trigger various events based on user interaction
```

Comment:
change to "Brizy's control button features a panel button that can trigger click event based on user interaction"

because "various events" means that can be more events, but actually we have only `onClick`